content/authors/adkris/_index.md

```diff
@@ -20,12 +20,12 @@ organizations:
 - name: Applied Physics Lab
 
 # Short bio (displayed in user profile at end of posts)
-bio: I am interested in applying machine learning algorithms for understanding echolocating animals
+bio: I am interested in studying echolocation as a sensory tool.
```

Comment:
"Sensor tool" is a little odd. What do you mean to say?

Reply:
I guess that part is also redundant but I wanted to explain that I am interested in studying how animals echolocate. How is that "tool" of echolocation being used by bats, for example, for their tasks

Reply:
I think "sensing tool" is more broadly used to describe this.

Reply:
I like "sensing tool" thanks!